examples/resources/ekscluster/cluster.tf

```diff
@@ -49,6 +49,23 @@ resource "tanzu-mission-control_ekscluster" "tf_eks_cluster" {
           "subnet-06897e1063cc0cf4e",
         ]
       }
+
+      addons_config {
+        vpc_cni_config {
+          eni_config {
+            id = "subnet-0a680171b6330619f" // Required, should belong to the same VPC as the cluster
+            security_groups = [
+              "sg-00c96ad9d02a22522",
```

Comment:
Example should include an empty security group list as well

Reply:
@DheerajSShetty  fixed this.